charts/gmsa/templates/deployment.yaml

```diff
@@ -56,6 +56,8 @@ spec:
           {{- if .Values.securityContext }}
           securityContext: {{ toYaml .Values.securityContext | nindent 12 }}
           {{- end }}
+          args:
+            - --cert-reload={{ .Values.certificates.certReload.enabled }}
```

Comment:
I am not familiar with where this gets used but I am assuming it will always be with an image with a binary that has `--cert-reload` option.

Reply:
Yes, Since the helm chart will be versioned I thought it was safe to set this here.